vdr/management/management.go

```diff
@@ -39,6 +39,9 @@ type DocumentManager interface {
 	DocumentOwner
 	resolver.DIDResolver
 
+	// Deactivate deactivates a DID document, making it unusable for future interactions.
+	Deactivate(ctx context.Context, id did.DID) error
+
```

Comment:
`DocManager` only needs `AddVerificationMethod` and `RemoveVerificationMethod` before it completely implements  `DocManipulator`. Both sound like they should be added for did:web?

Reply:
I wonder if 2 interfaces are needed?